### PR TITLE
Remove customer-visible access to Manifest Trust

### PR DESCRIPTION
**Pull request by gauravIoTEdge**
This PR removes:

- the manifest trust sample
- manifest trust related docs
- code coverage settings and project reference related to manifest trust

- [X] I have read the [contribution guidelines](https://github.com/azure/iotedge#contributing).

### CODE REVIEW

**Review by darobs — APPROVED** [review]
Make sure E2E test pass before it's merged. otherwise, looks fine.

File: edgelet/iotedge/src/config/apply.rs [hunk 1]
```diff
@@ -428,13 +428,18 @@ fn execute_inner(
...
     );
 
-    let manifest_trust_bundle_cert = manifest_trust_bundle_cert.map(|manifest_trust_bundle_cert| {
+    let manifest_trust_bundle_cert = None;
+    
+    // Skip Manifest trust 
+    /*
+    manifest_trust_bundle_cert.map(|manifest_trust_bundle_cert| {
         certd_config.preloaded_certs.insert(
             edgelet_settings::MANIFEST_TRUST_BUNDLE_ALIAS.to_owned(),
```
Comment (ggjjj) [inline edgelet/iotedge/src/config/apply.rs:437]: you can remove the whole comment

**Review by ggjjj — COMMENTED** [review]
Remove these two which are used by the manifest signer client 

https://github.com/gauravIoTEdge/iotedge/tree/removeSample/edge-util/src/es6numberserializer

https://github.com/gauravIoTEdge/iotedge/tree/removeSample/edge-util/src/jsoncanonicalizer

**Comment by gauravIoTEdge** [comment]
> Remove these two which are used by the manifest signer client
> 
> https://github.com/gauravIoTEdge/iotedge/tree/removeSample/edge-util/src/es6numberserializer
> 
> https://github.com/gauravIoTEdge/iotedge/tree/removeSample/edge-util/src/jsoncanonicalizer

@ggjjj - I'm going to leave these in -- they're not customer facing and we don't need to remove them. they're also referenced in a lot of places, and I don't think it's really necessary to remove them right now.

**Comment by gauravIoTEdge** [comment]
Building rust on windows was gnarly; building .NET on linux was gnarly.

This just turned into a one-stop-shop to make sure builds were passing, due to platform incompatibilities mentioned above.

Shall discard this PR and open a new one for wase of review.

**Pull request closed without merging**